script/git-clone-others.sh

```diff
@@ -117,6 +119,16 @@ for repository in $REPOSITORY_LIST ; do
                 repoAdditionalGitOptions=( "-b" "7.x" "${repoAdditionalGitOptions[@]}" )
             fi
         fi
+        if [ $(echo "$REPOSITORY_MAIN_LIST" | grep "^$repository$") ] ; then
```

Comment:
This works only if the branch is the first argument in `additionalGitOptions`, right? But I see it is already being assumed in the BRANCHED_7_REPOSITORY_LIST case, so probably not a problem right now.

Reply:
what if we rename now all branches to main to avoid struggling with this branch mapping ? it should be straightforward, we anyway need to rename it to main later or sooner